client/Makefile

```diff
@@ -73,15 +69,18 @@ e2e-annotations:
 
 .PHONY: smoke-test
 smoke-test:
-	start_server_and_test '$(MAKE) start-server' $(CXG_SERVER_PORT) '$(MAKE) e2e'
+	start_server_and_test \
```

Comment:
super tiny nit: I would make the `smoke-test` target test the default configuration (ie, annotations on), and change the other target to `smoke-test-disabled-annotations`

CC: @mweiden 

IMHO, this can be done in another PR unless it is easy/quick.

Reply:
Yeah, we probably need to chase this in a separate PR as making `smoke-test` have all default tests doesn't mesh well with the way the smoke tests tests are currently packaged.